src/amcrest/special.py

```diff
@@ -153,3 +155,62 @@ def mjpg_stream(self, channelno=None, typeno=None, path_file=None):
                 raise CommError(error)
 
         return ret.raw
+
+    def get_event_stream(self, eventcode='VideoMotion', callback=None,
```

Comment:
Actually now that I think about it a little more, and how this would be used, it doesn't seem to make a lot of sense to not allow a callback. At the very least I'd change the parameter list to put `callback` first without a default value. And then change the loop to just call the callback (and no option for returning.)

In fact, it might even make sense for the method to be a generator itself.